src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -530,16 +530,24 @@ public static synchronized EppoClient init(EppoClientConfig eppoClientConfig) {
                 eppoClientConfig.getBaseURL(),
                 Constants.REQUEST_TIMEOUT_MILLIS);
 
-        // Create wrapper for fetching experiment configuration
-        ExperimentConfigurationRequestor expConfigRequestor = new ExperimentConfigurationRequestor(eppoHttpClient);
-        // Create Caching for Experiment Configuration
+        // Create wrapper for fetching experiment and bandit configuration
+        ConfigurationRequestor<ExperimentConfigurationResponse> expConfigRequestor =
+          new ConfigurationRequestor<>(ExperimentConfigurationResponse.class, eppoHttpClient, Constants.RAC_ENDPOINT);
+        ConfigurationRequestor<BanditParametersResponse> banditParametersRequestor =
+          new ConfigurationRequestor<>(BanditParametersResponse.class, eppoHttpClient, Constants.BANDIT_ENDPOINT);
```

Comment:
Note: we decided to serve bandit parameters separately from the main RAC to not bloat it and to not have bandit parameter updating spoil CDN caching.